packages/framework/presence/src/presence.ts

```diff
@@ -30,6 +30,16 @@ import type { ISubscribable } from "@fluid-experimental/presence/internal/events
  */
 export type ClientSessionId = SessionId & { readonly ClientSessionId: "ClientSessionId" };
 
+/**
+ * The connection status of the {@link ISessionClient}.
+ *
+ * @alpha
+ */
+export enum SessionClientStatus {
+	Connected = "Connected",
+	Disconnected = "Disconnected",
+}
```

Comment:
Using enums in TypeScript is generally discouraged. We prefer objects instead unless there's a specific reason. I think they'll work fine here. <https://www.typescriptlang.org/docs/handbook/enums.html#objects-vs-enums>